### PR TITLE
Spelling corrections for multiple locations

### PR DESCRIPTION
**Pull request by jmartin-tech**
* Correct encoding in xss probe
* Consistent spelling in glitch probe
* Adjust README.md for consistency
* Adjust misp_descriptions for consistency

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -98,7 +98,7 @@ python3 -m garak --model_type huggingface --model_name gpt2 --probes dan.Dan_11_
 
 ## Reading the results
 
-For each probe loaded, garak will print a progress bar as it generates. Once generation is complete, a row evaluating that probe's results on each detector is given. If any of the prompt attempts yielded an undesirable behaviour, the response will be marked as FAIL, and the failure rate given.
```
Comment (erickgalinkin) [inline README.md:101]: British spelling is to be expected from @leondz :P

File: README.md [hunk 2]
```diff
@@ -192,14 +192,14 @@ For testing. This generator repeats back the prompt it received.
...
 | snowball | [Snowballed Hallucination](https://ofir.io/snowballed_hallucination.pdf) probes designed to make a model give a wrong answer to questions too complex for it to process |
-| xss | Look for vulnerabilities the permit or enact cross-site attacks, such as private data exfilteration. |
+| xss | Look for vulnerabilities the permit or enact cross-site attacks, such as private data exfiltration. |
 
 ## Logging
 
 `garak` generates multiple kinds of log:
 * A log file, `garak.log`. This includes debugging information from `garak` and its plugins, and is continued across runs.
 * A report of the current run, structured as JSONL. A new report file is created every time `garak` runs. The name of this file is output at the beginning and, if successful, also the end of the run. In the report, an entry is made for each probing attempt both as the generations are received, and again when they are evaluated; the entry's `status` attribute takes a constant from `garak.attempts` to describe what stage it was made at.
-* A hitlog, detailing attempts that yielded a vulnerability (a 'hit')
```
Comment (erickgalinkin) [inline README.md:202]: We safe the files as `{prefix}.hitlog.jsonl`, so I'm inclined to keep this spelling as a term of art, but you are technically correct -- the best kind of correct.

File: garak/probes/glitch.py [hunk 3]
```diff
@@ -1,7 +1,7 @@
 #!/usr/bin/env python3
 """Glitch keyword probes
 
-Testing for tokens that lead to unusual model behaviour
```
Comment (erickgalinkin) [inline garak/probes/glitch.py:4]: UK/US spelling again.

File: garak/resources/misp_descriptions.tsv [hunk 4]
```diff
@@ -59,8 +59,8 @@ avid-effect:performance:P0404	Environmental safety	Safety from environmental har
 quality:Behavioral:ContentSafety:Toxicity	Toxicity	Toxicity
 quality:Security:PromptStability	Prompt Stability	Prompt Stability
 quality:Robustness:GenerativeMisinformation	Generative Misinformation	Generative Misinformation
-quality:Behavioral:ContentSafety:HateHarassment	Hate and harassment	Hate and harrassment
+quality:Behavioral:ContentSafety:HateHarassment	Hate and harassment	Hate and harassment
 quality:Behavioral:ContentSafety:Profanity	Profanity	Profanity
-quality:Security:Confidentiailty	Confidentiailty	Confidentiailty
+quality:Security:Confidentiality	Confidentiality	Confidentiality
 quality:Behavioral:ContentSafety:HarmfulViolent	Harmful or violent	Harmful or violent
-quality:Behavioral:ContentSafety:Sexualized	Sexualised	Sexualised
```
Comment (erickgalinkin) [inline garak/resources/misp_descriptions.tsv:66]: Again, UK/US spelling

**Review by erickgalinkin — COMMENTED** [review]
Good catches overall -- I think there are a few things here that are a matter of UK vs. US spelling conventions _e.g._ behaviour that are fine to spell either way. Will defer to @leondz to accept the American way of life.

**Comment by jmartin-tech** [comment]
Note I made allowances for US vs UK spelling and and only made changes where consistency would be helpful.

File: README.md (hunk 2 above)
Comment (jmartin-tech) [reply README.md:202]: Consider then expanding this similar to the `log file` entry above as:

```
* A hit log, `{run_id}.hitlog.jsonl`, detailing attempts that yielded a vulnerability (a 'hit')
```

File: garak/probes/glitch.py (hunk 3 above)
Comment (jmartin-tech) [reply garak/probes/glitch.py:4]: Just for consistency as both spellings are in this file.

File: garak/resources/misp_descriptions.tsv (hunk 4 above)
Comment (jmartin-tech) [reply garak/resources/misp_descriptions.tsv:66]: Consistency as alternate spelling is used in in the same entry.

Comment (leondz) [reply garak/resources/misp_descriptions.tsv:66]: I only know English English and haven't put learning variants on my roadmap - go with whatever works for you :)

**Comment by leondz** [comment]
@erickgalinkin out of fear of being hanged for treason I'll defer to you to review/merge this

**Pull request merged**